NCATS_Data_Dictionary.md

```diff
@@ -808,93 +808,97 @@ document for [trustymail](https://github.com/cisagov/trustymail).
 
 ### rva Collection ###
 
-- `Summary`: Assessment Summary (ASMT\_ID / ASMT\_NAME)
-- `Status`:  Assessment Status (Open -\> Planning -\> Testing -\>
-  Reporting -\> Wrap Up -\> Completed)
-- `Created`:  Date ticket was created
-- `Updated`:  Last update date
-- `App A Date`:  Date Appendix A was signed
-- `App A Signed`:  Was appendix A signed? (Boolean)
-- `App B Signed`:  Date Appendix B was signed, if applicable
-- `Asmt Type`:Assessment Type (RVA, HVA, RPT, PCA, VADR)
-- `External Testing Begin`:  Date of beginning of external testing
-- `External Testing End`:  Date of end of external testing
-- `Group/Project`: Group or Project
-- `Internal Testing Begin`:  Date of beginning of internal testing
-- `Internal City`:  Location (City) of on-site testing if applicable
-- `Internal Testing End`:  Date of end of internal testing
-- `Mgmt Req`:  Management Request (DHS, NCCIC, EOP, FALSE)
-- `POC Email`:  POC E-mail address
-- `POC Name`:  POC Name
-- `POC Phone`:  POC Phone Number
-- `ROE Date`:  Date ROE is signed
-- `ROE Number`:  ROE Number (assigned by NCATS)
-- `ROE Signed`:  ROE Signed (Boolean)
-- `_ID`:  Assessment ID (RV0XXX for RVA/HVA/PCA or VR0XXX for VADR)
-- `Asmt Name`:  Assessment Name (Usually customer name and assessment
+- `_ID` [ObjectId]:  Assessment ID (RV0XXX for RVA/HVA/PCA or VR0XXX for VADR)
+- `App A Date` [ISO date]:  Date Appendix A was signed
```

Comment:
There is inconsistent capitalization of "Appendix" in this section.  Pick either `Appendix` or `appendix` and update accordingly.  @kitaf004, I know this is beyond the scope of what you were changing, but it caught my eye and should be fixed.